src/assets/styling/pages/_home.scss

```diff
@@ -287,13 +287,17 @@
   }
 }
 .dialog-close {
+  @include pxRem(right, 30);
+  @include respond-below(wide) {
+    @include pxRem(top, 30);
+  }
   z-index: 200;
   background-color: map-get($color, white);
   transition-duration: 0.25s;
   user-select: none;
   border-width: 0px;
   outline: none;
-  position: relative;
+  position: fixed;
```

Comment:
why ?

Reply:
because required want close button fixed in dialog screen.